manual/photon.xml.template

```diff
@@ -77,6 +77,14 @@
             <Label>vCenter Password</Label>
             <Description>Password to login to vCenter Server</Description>
         </Property>
+        <Property ovf:key="guestinfo.vcenter_veba_ui_username" ovf:type="string" ovf:userConfigurable="true" ovf:value="">
+            <Label>vCenter Username to register VEBA UI</Label>
+            <Description>Username to register VMware Event Broker UI to vCenter Server (Optional for Knative Processor)</Description>
```

Comment:
May I suggest slight reword to avoid ambiguity?

```suggestion
            <Description>Username to register VMware Event Broker UI to vCenter Server for Knative Processor (optional)</Description>
```

Reply:
Fixed